sdk/python/foundation-models/system/inference/text-to-image/image-text-to-image-batch-endpoint.ipynb

```diff
@@ -0,0 +1,520 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Image to Image Inference using Online Endpoints\n",
+    "\n",
+    "This sample shows how to deploy `image to image` type stable diffusion models to a batch endpoint for inference.\n",
+    "\n",
+    "### Task\n",
+    "`image to image` task takes an original image and a text prompt as input. The model generates an image by modifying the original image.\n",
+    "\n",
+    " \n",
+    "### Model\n",
+    "Models that can perform the `image to image` task are tagged with `image-text-to-image`. We will use the `stabilityai-stable-diffusion-xl-refiner-1-0` model in this notebook. If you opened this notebook from a specific model card, remember to replace the specific model name.\n",
```

Comment:
The task should be image to image. 
Have left the same comment on model asset PR - https://github.com/Azure/azureml-assets/pull/1785.